projects/ui-components/lf-repository-browser/lf-repository-browser.component.ts

```diff
@@ -70,6 +74,7 @@ export class LfRepositoryBrowserComponent implements OnDestroy {
   @Input()
   refreshAsync: () => Promise<void> = async () => {
     if (!this._currentFolder) {
+      this.isLoading = true;
```

Comment:
there is no place where isLoading is set to false?
or is that in a later call?

do we need to initialize breadcrumbs only if there was no currentFolder

This is a more general question, but if there is no currentFolder do we want to try to get the root, or just throw?

Reply:
yeah if there is no currentFolder, we try to get the root, but if there is still no currentFolder, we throw.
should we display error instead? (and add a try catch block for the getRootTreeNodeAsync call)